### DISCUSSION TITLE
opengraph-image.jpg in public folder

### DISCUSSION THREAD

**Discussion by AllmostHumann** [vercel/next.js#49503] [Help] [answered]
### Summary

Hi everyone,

I'm trying to set all the metadata in my application and I encountered an error (unless I'm doing something wrong) with the og:image implementation. According to the documentation, I can drop a file named "opengraph-image.jpg" somewhere in the "route segment". I tried leaving it in the public folder. And seemingly everything is ok because in devtools in I see , but in sharing debugger the image does not display. Only after uploading the file to /app did everything work.

Is this a bug or is this what I should do?

Ps: favicon.ico in public folder works perfectly.
Ps2: This is my first approach to next.js and I'm probably doing something wrong for which I apologize in advance.

Thanks! <3

### Additional information

```js
Error message at sharing debugger "Could not be processed as an image because it has an invalid content type" when the image is in public folder
```

**Comment by Nusab19** [comment]
**Same problem here**

**Comment by leerob** [comment, marked as answer]
The file does not go in the `public/` folder, but instead the `app/` folder 👍 

Secondly, ensure you are using `metadataBase` for your layout or page:

```js
export const metadata = {
  metadataBase: new URL('https://acme.com'),
}
```

**Reply by dm-xai** [reply]
Is there a way to use "symlink" to avoid putting same image, that is used across few places into different folders in repo?
